src/Features/CSharp/Portable/Completion/CompletionProviders/ObjectCreationCompletionProvider.cs

```diff
@@ -78,6 +78,11 @@ protected override (string displayText, string suffix, string insertionText) Get
                 return (symbol.Name, "", symbol.Name);
             }
 
+            if (symbol is INamespaceOrTypeSymbol namespaceOrTypeSymbol)
+            {
+                return base.GetDisplayAndSuffixAndInsertionText(namespaceOrTypeSymbol.WithoutNullability(), context);
```

Comment:
is this only top level, or does it also affect nested nullability?

Reply:
Top level only

Reply:
Top level only

Reply:
Unless you want to write this WithNullability(NullableAnnotation.NotAnnotated), the difference being that once the compiler internalizes the wrapping, WithoutNullability() is an extension just going away, whereas "no, I'm actually changing the symbol" is the real intent here.

Reply:
(and you used this pattern in the extract method thing as well for this same reason)